### PR TITLE
cards added

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Screenshot:
card
## Deploy Preview Link:  
https://deploy-preview-10--comp-ui.netlify.app/components/card/card.html

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+:root{
+    --card-primary-color: #2a9d8f;
+    --card-font-color : #fff;
+    --card-border-color : #bacdd8;
+    --card-dark-color : #000;
+    --card-status-color : #807F7F;
+}
+.card-container {
+    max-width: 230px;
+    min-width: 220px;
```
Comment (akash-1318) [inline components/card/card.css]: You should not use pixel here.

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    opacity: 0.3;
+}
+.card-status-content{
+    text-align: center;
+}
+
+.card-horizontal-container{
+    display: flex;
+    max-width: 430px;
+    min-width: 220px;
```
Comment (akash-1318) [inline components/card/card.css]: don't use pixel.

File: components/card/card.css (hunk 1 above)
Comment (tushargoyal812) [reply components/card/card.css]: yes don't use pixels it will create problems in responsiveness.

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    cursor: pointer;
+}
+.text-striked{
+    text-decoration: line-through;
+    font-size: 0.8em;
+    color: var(--card-status-color);
+}
+
+.card-status{
+    font-size: 2rem;
```
Comment (tushargoyal812) [inline components/card/card.css]: use only one unit like if you using rem then use it everywhere and rem is best practice for responsive design.

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    color: var(--card-dark-color);
+    border-radius: 0.1em;
```
Comment (tushargoyal812) [inline components/card/card.css:74]: suggestion: as vikram said use px for borders and border radius.

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+}
+.card-horizontal{
+    display: flex;
+}
+.card-col{
+    max-width: 50%;
+    position: relative;
+    background-color: var(--card-font-color);
+    padding: 4px;
```
Comment (tushargoyal812) [inline components/card/card.css]: don't use pixels for padding

**Review by tushargoyal812 — COMMENTED** [review]
take care of some suggestions and looks good to marge.

File: components/card/card.css (hunk 1 above)
Comment (bishnoimukesh) [reply components/card/card.css]: changed to em

File: components/card/card.css (hunk 2 above)
Comment (bishnoimukesh) [reply components/card/card.css]: changed to em

File: components/card/card.css (hunk 3 above)
Comment (bishnoimukesh) [reply components/card/card.css]: done

File: components/card/card.css (hunk 5 above)
Comment (bishnoimukesh) [reply components/card/card.css]: done

**Pull request merged**